emission/net/api/cfc_webapp.py

```diff
@@ -388,32 +389,59 @@ def getCarbonCompare():
 @get('/eval/publicData/timeseries')
 def getPublicData():
   ids = request.json['phone_ids']
-  uuids = map(lambda id: UUID(id), ids)
-
+  uuids = map(lambda id: __getUUIDFromEmail__(id), ids)
+  
   from_ts = request.query.from_ts
   to_ts = request.query.to_ts
 
-  time_range = estt.TimeQuery("metadata.write_ts", float(from_ts), float(to_ts))
-  time_query = time_range.get_query()
-  
+  public_queries = map(lambda id: {'user_id': id, "metadata.key":  "eval/public_device"}, uuids)
+  public_ranges = map(list(edb.get_timeseries_db().find(q))["data"], public_queries)
```

Comment:
I would recommend using functions from `emission.storage.timeseries` instead of rewriting your own queries. It will be easier to switch to other timeseries (e.g BtrDB) if we have a clearly defined interface that all modules use.